enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult/InteractiveQuestionResult.tsx

```diff
@@ -76,7 +76,7 @@ export const InteractiveQuestionResult = ({
   }
 
   // `isCardError: true` when the entity ID couldn't be resolved
-  if ((!question || isCardIdError) && originalId) {
+  if ((!question || isCardIdError) && originalId && originalId !== "new") {
```

Comment:
did you miss [line 62 from the original PR](https://github.com/metabase/metabase/pull/54323/files#diff-1343abc72f5ab802afc2d06a80a61cdec025e08de0623ebb13b60dbd4c85a3d0R62)? shouldn't it be `originalId === "new"` too?

Reply:
updated, thanks!